packages/react/menu/src/Menu.tsx

```diff
@@ -360,8 +356,7 @@ type MenuLabelPrimitive = Polymorphic.ForwardRefComponent<
 >;
 
 const MenuLabel = React.forwardRef((props, forwardedRef) => {
-  const { selector = getSelector(LABEL_NAME), ...labelProps } = props;
-  return <Primitive {...labelProps} selector={selector} ref={forwardedRef} />;
+  return <Primitive {...props} ref={forwardedRef} />;
 }) as MenuLabelPrimitive;
```

Comment:
Looks like this one could just be an `extendComponent` now?

```jsx
const MenuLabel = extendComponent(Primitive, LABEL_NAME);
```

I've always wondered if we really need this part tho tbh... it doesn't provide any accessibility, semantics or functionality 🤔

Reply:
oh good point, I didn't pay attention to that! Do you think others became like that too?
Hmm, actually probably not, this is just because that one was already possible to use `extendComponent`.

> I've always wondered if we really need this part tho tbh... it doesn't provide any accessibility, semantics or functionality 🤔

Regarding this, I think it's mostly an expectation of what these components offer, we can always re-assess separately though.

Reply:
Addressed.